cmd/up.go

```diff
@@ -131,15 +132,10 @@ func newUpCmd() *cobra.Command {
 			contract.IgnoreError(os.RemoveAll(temp))
 		}()
 
-		// Cleanup the project name/description if needed.
-		projectName := template.ProjectName
-		if projectName == "${PROJECT}" {
-			projectName = workspace.ValueOrSanitizedDefaultProjectName(projectName, template.Name)
-		}
-		projectDescription := template.ProjectDescription
-		if projectDescription == "${DESCRIPTION}" {
-			projectDescription = ""
-		}
+		// Get the project name/description.
+		projectName := workspace.ValueOrSanitizedDefaultProjectName("", template.ProjectName, template.Name)
+		projectDescription := workspace.ValueOrDefaultProjectDescription(
+			"", template.ProjectDescription, template.Description)
```

Comment:
Just a note about `up`'s behavior here: Right now it always uses these values automatically without prompting the user. That behavior isn't correct as noted in #1871. This PR maintains the existing behavior. The fix for #1871 will update `up` to prompt as needed.